raw_data/preprocessed/africa/morocco_french.txt

```diff
@@ -0,0 +1,219 @@
+Chapitre Premier - Dispositions générales 
+
+Section Première - Objectifs et principes généraux 
+
+(1) La présente loi a pour objet d'édicter les règles de base et les principes généraux de la politique nationale dans le domaine de la protection et de la mise en valeur de l'environnement. Ces règles et principes visent à: - protéger l'environnement contre toutes formes de pollution et de dégradation quelle qu'en soit l'origine; - améliorer le cadre et les conditions de vie de l'homme; - définir les orientations de base du cadre législatif, technique et financier concernant la protection et la gestion de l'environnement; - mettre en place un régime spécifique de responsabilité garantissant la réparation des dommages causés à l'environnement et l'indemnisation des victimes.
+
+(2) L'application des dispositions de la présente loi se base sur les principes généraux suivants: [•] La protection, la mise en valeur et la bonne gestion de l'environnement font partie de la politique intégrée du développement économique, social et culturel; [•] La protection et la mise en valeur de l'environnement constituent une utilité publique et une responsabilité collective nécessitant la participation, l'information et la détermination des responsabilités; [•] L'instauration d'un équilibre nécessaire entre les exigences du développement national et celles de la protection de l'environnement lors de l'élaboration des plans sectoriels de développement et l'intégration du concept du développement durable lors de l'élaboration et de l'exécution de ces plans; [•] La prise en considération de la protection de l'environnement et de l'équilibre écologique lors de l'élaboration et de l'exécution des plans d'aménagement du territoire; [•] La mise en application effective des principes de " l'usager payeur " et " du pollueur payeur " en ce qui concerne la réalisation et la gestion des projets économiques et sociaux et la prestation de services; [•] Le respect des pactes internationaux en matière d'environnement lors de l'élaboration aussi bien des plans et programmes de développement que de la législation environnementale.
```

Comment:
Please remove the unnecessary spacing around the double quotation marks in `" l'usager payeur "` and `" du pollueur payeur "`.